cmd/minishift/cmd/start.go

```diff
@@ -215,16 +216,25 @@ func handleProxies() *util.ProxyConfig {
 	return proxyConfig
 }
 
+func getSlice(key string) []string {
+	viper.Set(startFlags.InsecureRegistry.Name, []string{"172.30.0.0/16"})
```

Comment:
That seems fugly. You are calling a getter, but internally the whole impl has side effects, since it calls a viper.Set as well. 

Also you are setting the value for keys which are completely unrelated to _startFlags.InsecureRegistry.Name_,  not to mention that you do it multiple times.

I think this needs to be handled explicitly.